.github/actions/go-aio/action.yaml

```diff
@@ -12,15 +12,20 @@ inputs:
     default: ""
   version:
     description: "Go version"
-    required: true
+    required: false
     default: "1.18"
+  version-file:
+    description: "Go version file (go.mod)"
+    required: false
+    default: ""
 
 runs:
   using: "composite"
   steps:
     - uses: actions/setup-go@v3
```

Comment:
Looks like the latest version is v5 - https://github.com/actions/setup-go

Reply:
I noticed that, but don't want to break things unknowingly.

Reply:
Also I have checked and this `go-version-fil` param was supported on `v3`.

Reply:
Hmm I'd recommend a bump to at least v4, where caching is enabled by default. We already use it in many places and I don't _think_ it should break things.

Reply:
Good point. Will do.